library/core/src/lib.rs

```diff
@@ -310,6 +310,7 @@ pub mod ops;
 pub mod any;
 pub mod array;
 pub mod ascii;
+pub mod asserting;
```

Comment:
Wouldn't this need an `#[unstable]` attribute like the `async_iter` below it?

This looks to me like it would allow people to `use core::asserting;` even though nothing in there is stable?

Reply:
There's a `#![unstable]` in the module, so I think it shouldn't be needed.